test/import.bats

```diff
@@ -390,6 +390,79 @@ eigth:
     [ -f /dir/file2 ]
     [ -f /dir/files/file3 ]
 EOF
+    stacker build
+}
+
+@test "import with dir contents" {
+  mkdir folder1
+  touch folder1/file1
+  mkdir folder1/subfolder2
+  touch folder1/subfolder2/subfile1
+  cat > stacker.yaml <<EOF
+first:
+  from:
+    type: oci
+    url: $CENTOS_OCI
+  import:
+  - path: folder1/
+    dest: /folder1/
+  run: |
+    [ -f /folder1/file1 ]
+
+second:
+  from:
+    type: built
+    tag: first
+  run: |
+    mkdir -p /folder1
+    touch /folder1/file1
+    touch /folder1/file2
+
+third:
+  from:
+    type: oci
+    url: $CENTOS_OCI
+  import:
+    - path: stacker://second/folder1/
+      dest: /folder1/
+  run: |
+    [ -f /folder1/file1 ]
+    [ -f /folder1/file2 ]
 
+fourth:
+  from:
+    type: oci
+    url: $CENTOS_OCI
+  import:
+  - path: folder1/
+    dest: /
+  run: |
+    ls -l /
+    [ -f /file1 ]
+    mkdir -p /folder4/subfolder5
+    touch /folder4/subfolder5/subfile6
+
+fifth:
+  from:
+    type: oci
+    url: $CENTOS_OCI
+  import:
+  - path: folder1/subfolder2/
+    dest: /folder3/
+  - path: folder1/subfolder2
+    dest: /folder4/
+  - path: stacker://fourth/folder4/subfolder5/
+    dest: /folder6/
+  - path: stacker://fourth/folder4/
+    dest: /folder7/
```

Comment:
You are not testing an import of a 'stacker://' path without a trailing /.

You should add an import of:
```yaml
 - path: stacker://fourth/folder4
   dest: /folder8
```

And then a test of 
```bash
  [ -d /folder8/folder4 ]
```

Reply:
The issue is if folder8 is present then [ -d /folder8/folder4 ] will evaluate to true else false. But let me add this test and see what happens.

Reply:
oh, yuck.

i think that 'target' (the full path to the imported thing) should be fully determined by the 'dest', and not dependent on either the 'path' or the contents of the filesystem to which it is being imported.

that just seems the least surprising behavior to me.

Reply:
Removed the test.